tests/framework/installer/ceph_manifests.go

```diff
@@ -58,6 +58,7 @@ type ClusterSettings struct {
 	RBDMirrorWorkers int
 	UsePVCs          bool
 	StorageClassName string
+	withoutOSDs      bool
```

Comment:
I'd suggest naming this `skipOSDCreation`

Reply:
ok